application/nodejs/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "nodejs-secrethub-demo",
+  "version": "1.0.0",
+  "description": "nodejs-secrethub-demo",
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js"
+  },
+  "author": "SecretHub",
+  "license": "ISC",
```

Comment:
I'd say let's do `MIT`.

Reply:
Sure, I'll change it.

Reply:
And the comment about the package-json.lock, if it does not exist, there will be a warning. Another option is to touch the file with Docker, but I thought this was a nice solution.

Reply:
NPM should automatically populate the `package-lock.json`, didn't it do that? Or did you remove it?

Reply:
Yes you're right. npm does create automaticly. I'll remove package-lock.json

Reply:
No I meant: include it. Because it increases portability.

Reply:
Ah, so include it and leave it empty like it was?

Reply:
No, not empty, but populated with the contents it autogenerates when you run `npm install`.

Reply:
Sure, I'll add it